packages/smooth_app/lib/pages/onboarding/country_selector.dart

```diff
@@ -3,11 +3,18 @@ import 'package:iso_countries/iso_countries.dart';
 import 'package:openfoodfacts/utils/CountryHelper.dart';
 import 'package:smooth_app/data_models/user_preferences.dart';
 import 'package:smooth_app/database/product_query.dart';
-import 'package:smooth_ui_library/util/ui_helpers.dart';
 
 /// A selector for selecting user's country.
 class CountrySelector extends StatefulWidget {
-  const CountrySelector();
+  const CountrySelector({
+    this.initialCountryCode,
+    this.inputDecoration,
+    this.padding,
+  });
+
+  final String? initialCountryCode;
```

Comment:
consider making this required and set to `WidgetsBinding.instance?.window.locale.countryCode?.toLowerCase()` from welcome_page. Kind of makes sense to specify the initial country code and the reader doesn't have to read the internal code of this class to find out what happens if this parameter is not set

Reply:
Good point!